test/support/vm/openldap/README.md

```diff
@@ -27,6 +46,10 @@ $ export INTEGRATION_HOST=$ip
 
 # now run tests without having to set ENV variables
 $ time bundle exec rake
+
+# Once you're all done
+$ cd test/support/vm/openldap
+$ vagrant destroy
 ```
 
 You may need to `gem install vagrant` first in order to provision the VM.
```

Comment:
Earlier in this file, it lists that the vagrant gem is no longer supported:

> NOTE: The Vagrant gem (gem install vagrant) is no longer supported

Perhaps it would be good to remove this to avoid confusion?